### DISCUSSION TITLE
How to use the updater for dev version?

### DISCUSSION THREAD

**Discussion by math-GH** [FreshRSS/FreshRSS#3781] [Show and tell]
FreshRSS has a built-in updater (that is very useful!) to update to the newest stable version.

Is it also possible to switch to the development "channel" ("edge" branch)?

As I read https://github.com/FreshRSS/update.freshrss.org/blob/master/scripts/update_to_dev.php it should be possible, but I cannot find how to switch.

**Comment by Frenzie** [comment]
I think there's a file somewhere, although I can't seem to find it. Maybe the method was changed.

**Comment by math-GH** [comment]
found it:
https://github.com/FreshRSS/FreshRSS/blob/6004de845fb2d235a75ccc516727dd8181ad5e71/constants.php#L6
it is already set when you install from the edge branch

**Reply by Frenzie** [reply]
I don't think that's what I mean but if that's the way to go these days. :+1: 

Edit: the principle's the same anyway, adding `-dev` at the end.

**Reply by oliversum-ch** [reply]
@Frenzie how exactly did you do that? I'm also on 1.19.0-dev., and the updater says that there's no new version, although 1.19.2 is ready.

**Reply by Frenzie** [reply]
I don't think you mean me, but note the linked line.

**Comment by Alkarex** [comment]
There is an old PR, which I never completed, to rewrite the update mechanism https://github.com/FreshRSS/FreshRSS/pull/1760
At the moment, the best way to change branch (e.g., from *latest* = *stable* to *edge* = *dev*) is via git or Docker.
